File: src/coralsarticle/benchmark/algorithms/julia/exps.jl
using Corals
include("./registry/cor.jl")
include("./registry/topk.jl")

function load_experiments(X; kwargs...)

    experiments  = Dict()

    merge!(experiments, load_experiments_cor(X; kwargs...))
    merge!(experiments, load_experiments_topk(X; kwargs...))

    return experiments
end

File: src/coralsarticle/benchmark/algorithms/r/exps.R
root_dir <- "./src/coralsarticle/benchmark/algorithms/r"
source(paste(root_dir, "registry/cor.R", sep="/"))
source(paste(root_dir, "registry/topk.R", sep="/"))
load_experiments <- function(name, d, ...) {

    exp = load_experiments_cor(name, d, ...)
    if (is.null(exp)) {
        exp = load_experiments_topk(name, d, ...)
    }
    return(exp)
}